SuiteSparse_config/Makefile

```diff
@@ -41,11 +41,11 @@ library:
 
 # install only in SuiteSparse/lib and SuiteSparse/include
 local:
-	( cd build && cmake $(CMAKE_OPTIONS) -DLOCAL_INSTALL=1 .. && cmake --build . --config Release -j${JOBS} )
+	( cd build && cmake $(CMAKE_OPTIONS) -USUITESPARSE_PKGFILEDIR -DLOCAL_INSTALL=1 .. && cmake --build . --config Release -j${JOBS} )
 
 # install only in /usr/local (default)
 global:
-	( cd build && cmake $(CMAKE_OPTIONS) -DLOCAL_INSTALL=0 .. && cmake --build . --config Release -j${JOBS} )
+	( cd build && cmake $(CMAKE_OPTIONS) -USUITESPARSE_PKGFILEDIR -DLOCAL_INSTALL=0 .. && cmake --build . --config Release -j${JOBS} )
```

Comment:
It would be surprising to me as a user if the build rules would undefine a flag that I set in `CMAKE_OPTIONS`.

Would it be better to undefine that flag *before* adding the flags from `CMAKE_OPTIONS`?

Reply:
I had trouble with this variable getting "stuck" in one mode and not getting changed to the correct value with "make local".